crates/iceberg/src/spec/manifest.rs

```diff
@@ -1328,7 +1334,7 @@ mod _serde {
             Ok(Self {
                 content: value.content as i32,
                 file_path: value.file_path,
-                file_format: value.file_format.to_string(),
+                file_format: value.file_format.to_string().to_ascii_uppercase(),
```

Comment:
Pyiceberg fail to read if this file format is not uppercase. BTW, we can add test using pyiceberg to ensure the write is valid for other SDK later.

Reply:
Good catch, I've created an issue to get that fixed: https://github.com/apache/iceberg-python/issues/1340